spec/openapi.yaml

```diff
@@ -181,6 +191,7 @@ paths:
     get:
```

Comment:
Do we want `/listing_versions` or `/listing/<id>/versions`? To me, a version only makes sense as a part of a listing, so I would expect it to be a sub-resource.

Reply:
Also, where would I post a new version to, and how would I associate it with its listing?

Reply:
For the association, there is a listing_id FK that makes the association for us. This information is included when you request a listing, but is written separately so that you don't need to post the entire listing to delete a version. The URL doesn't make an enormous difference in the end, it's just far cleaner to have it be a separate path. I can talk to Chris to see if that makes sense on our end

Reply:
I would be open to support both.